### PR TITLE
Adjust help labels for the Keylime roles

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

File: control/control.MicroOS.xml [hunk 1]
```diff
@@ -586,7 +587,7 @@ configured accordingly to match the use case of the role. &lt;/p&gt;</label>
         <micro_os_role_ra_verifier_description>
 	<label>• Same installation as MicroOS role
 • Remote attestation verifier based on Keylime and TPM2.0 (required hardware)
-• Create /etc/keylime.conf based on /etc/usr/keylime.conf and start keylime_{verifier,registrar}.service</label>
+• Start keylime_{verifier,registrar}.service</label>
```
Comment (Vogtinator) [inline control/control.MicroOS.xml]:
```
• Start keylime_{verifier,registrar}.service</label>
```

That's sh syntax, not user parsable

Comment (aplanas) [reply control/control.MicroOS.xml]: Done

**Review by Vogtinator — APPROVED** [review]

**Comment by dgdavid** [comment]
Ouch! Sorry for the delay, somehow I overlooked the approval… until now. Too busy lately :) Merging it.

**Review by dgdavid — APPROVED** [review]
Approved by @Vogtinator on Oct 10

**Pull request merged**